Fitness/graphql/gymClassInstanceQueries.graphql

```diff
@@ -10,18 +9,20 @@ query AllClassesInstances
         id
         name
         description
-        tags
+          tags {
+              label
+          }
       }
+      imageUrl
       startTime
       endTime
       instructor
       isCancelled
     }
   }
 
-query TodaysClasses($dateTime: DateTime)
-{
-    classes(now: $dateTime) {
+query TodaysClasses($date: Date) {
```

Comment:
Shouldn't the graphql type be `DateTime` and not `Date`?

Reply:
Kenneth moved it to Date